setup.py

```diff
@@ -2,8 +2,8 @@
 import versioneer
 
 install_requires = ['menpo>=0.7,<0.8',
-                    'cyrasterize>=0.2.4,<0.3',
-                    'mayavi>=4.4.4']
+                    'cyrasterize>=0.3,<0.4',
+                    'mayavi>=4.5.0']
```

Comment:
@patricksnape doesn't have to be no, was mainly because users have reported issues with 4.4.4 that have hopefully been resolved in 4.5.0. Want us to loosen it? 4.5.0 is on pip and our channel so we should be good for the dep...I'm easy.